site/src/components/roadmap/Roadmap.tsx

```diff
@@ -0,0 +1,220 @@
+import { ReactNode, SetStateAction, useEffect, useState } from "react";
+import {
+  GridLayout,
+  Card,
+  CardProps,
+  Input,
+  Link,
+  Banner,
+  BannerContent,
+} from "@salt-ds/core";
+import styles from "./style.module.css";
+import {
+  ProgressInprogressIcon,
+  ProgressPendingIcon,
+  SearchIcon,
+} from "@salt-ds/icons";
+import { Heading3 } from "../mdx/h3";
+import { formatDate } from "src/utils/formatDate";
+import { Heading2 } from "../mdx/h2";
+
+type RoadmapProps = { title: string; children: ReactNode };
+
+interface RoadmapData {
+  id: string;
+  startDate: Date;
+  targetDate: Date;
+  issueUrl: string;
+  text: string;
+}
+
+type RoadmapDataItem = {
+  text: string;
+  startDate: string;
+  targetDate: string;
+  status: string;
+  issueUrl: string;
+};
+
+interface CardViewProps {
+  sortedRoadmapData: RoadmapData[];
+  searchQuery: string;
+}
+
+function sortRoadmapDataByDate(roadmapData: RoadmapData[]): RoadmapData[] {
+  const sortedData = [...roadmapData];
+  sortedData.sort((a, b) => {
+    const startDateA = new Date(a.targetDate);
+    const startDateB = new Date(b.targetDate);
+
+    if (isNaN(startDateA.getTime())) {
+      return 1; // Item A doesn't have a valid date, so it should be placed below item B
+    } else if (isNaN(startDateB.getTime())) {
+      return -1; // Item B doesn't have a valid date, so it should be placed below item A
+    } else {
+      return startDateA.getTime() - startDateB.getTime();
+    }
+  });
+  return sortedData;
+}
+
+function RoadmapCard(props: CardProps) {
+  return (
+    <Card
+      className={styles.spacing}
+      style={{ width: "250px", marginBottom: "15px", minHeight: "unset" }}
```

Comment:
I would consider if we need these to be responsive.
 Inline styles cannot be overriden, so would have this driven by className.
 Can the size be driven by the container or theme ?
 At the very least we could create css vars for these values, so they can be overriden.
You are passing a className in props and that is overridding the classname on line 64, so that might be why you had to add an inline style.